user-guide.rst

```diff
@@ -4,24 +4,28 @@ User Guide & FAQ
 Frequently Asked Questions
 --------------------------
 
-.. rubric:: Why are there different Scenario Explorer for specific projects?
+.. rubric:: Why are there different *Scenario Explorer* instances for specific projects?
```

Comment:
The word _instance_ might be too technical here, I'd just use the plural, Scenario Explorers:


```suggestion
.. rubric:: Why are there different Scenario Explorers for specific projects?
```